batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/controller/StudentController.java

```diff
@@ -0,0 +1,201 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.service.StudentService;
+import com.lftechnology.batch7crud.util.TypeCaster;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * @Author binodnme
+ * Created on 1/14/16
+ */
+@WebServlet("/students/*")
+public class StudentController extends HttpServlet{
+    StudentService studentService = new StudentService();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException{
+        String servletPath = req.getPathInfo();
+
+        if(servletPath == null || "/".equalsIgnoreCase(servletPath)){
+            Integer page = 1;
+            try{
+                page = Integer.parseInt(req.getParameter("page"));
+            }catch (NumberFormatException e){}
+
+            list(req, resp, page);
+
+        }else{
+            String[] params = servletPath.split("/");
+
+            if(params[1].equalsIgnoreCase("create")){
+                create(req, resp);
+
+            } else if(params[2].equalsIgnoreCase("edit")){
+                Integer studentId = TypeCaster.toInt(params[1]);
+                edit(req, resp,studentId);
+            }
+        }
+
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String servletPath = req.getPathInfo();
+        String[] params = servletPath.split("/");
+
+        if(params[1].equalsIgnoreCase("create")){
+            createProcess(req,resp);
+
+        }else if(params[2].equalsIgnoreCase("edit")){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "edit" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)